mne/io/kit/constants.py

```diff
@@ -71,6 +71,13 @@
 KIT_NY.LPFS = [10, 20, 50, 100, 200, 500, 1000, 2000]
 
 
+# Maryland-system channel information
+# Virtually the same as the NY-system except new ADC circa July 2014
+# 16-bit A-to-D converter, one bit for signed integer. range +/- 32,768
+KIT_MD = Bunch(**KIT_NY)
+KIT_MD.DYNAMIC_RANGE = 2 ** 16 / 2
```

Comment:
on Python 2 this will be integer division and on Python 3 it will be float, please make explicit with `2 ** 16 // 2` (or really just `2 ** 15` would be fine) or `float(2 ** 15)` depending on what you want the output type to be